client/my-sites/plugins/marketplace/marketplace-plugin-upsell.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import QueryProducts from 'calypso/components/data/query-products-list';
+import { getProductCost, isProductsListFetching } from 'calypso/state/products-list/selectors';
+import { MARKETPLACE_YOAST } from '@automattic/calypso-products';
+import formatCurrency from '@automattic/format-currency';
+import { getCurrentUserCurrencyCode } from 'calypso/state/current-user/selectors';
+import { Button } from '@automattic/components';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function MarketplacePluginUpsell( props ) {
+	const cost = useSelector( ( state ) => getProductCost( state, MARKETPLACE_YOAST ) );
+	const currencyCode = useSelector( ( state ) => getCurrentUserCurrencyCode( state ) );
+	const isLoading = useSelector( ( state ) => isProductsListFetching( state ) );
+	const displayCost = formatCurrency( cost, currencyCode, { stripZeros: true } );
+	const { translate } = props;
+
+	return (
+		<div className="marketplace-plugin-upsell">
+			<QueryProducts />
+			<h3>{ translate( 'Premium' ) }</h3>
+			<div class="marketplace-plugin-upsell__description">
+				{ translate( 'Features included:' ) }
+			</div>
+			<ul>
+				<li>
+					<Gridicon size={ 12 } icon="checkmark" />
+					{ translate( '.blog custom domain included' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Custom domain not included' )` *ES Score: 6*